web/README.md

```diff
@@ -56,4 +56,11 @@ If more than one target is specified, the last one will take precedence.
 
 Before running unit tests on KeymanWeb, first run `./build.sh` according to the instructions above.
 
-Once the build is complete, running `npm test` will run the unit testing suite on your local machine in-browser.  Alternatively, see `unit_tests/test.sh`, which the former command executes.
+Once the build is complete, running `npm test` will run the unit testing suite on your local machine in-browser.
+Alternatively, see `unit_tests/test.sh`, which the former command executes.
+
+### Debugging Unit Test
+1. During development, to run a specific unit test, change the `it` to `it.only`.
```

Comment:
Can also be done for a grouping of tests:  `describe.only` will run all tests under that group.